wayfinder/CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+# CHANGELOG - wayfinder
+
+## 1.0.0 / 2023-06-12
+* [Added] Initial Release
```

Comment:
```suggestion
# Changelog - Wayfinder

## 1.0.0
***Added:***

* Initial release
```
Updating to match new standard formatting for changelogs

Reply:
updated